benchmarks/base64/CMakeLists.txt

```diff
@@ -9,6 +9,8 @@ CPMAddPackage(
 
 add_executable(benchmark_base64 benchmark_base64.cpp)
 
-target_link_libraries(benchmark_base64 PUBLIC simdutf)
+set_property(TARGET benchmark_base64 PROPERTY CXX_STANDARD 17)
```

Comment:
Is this a semver-major change?

Reply:
It will be but the benchmarks are optional. We only make sure that the core library is C++11.

Reply:
I will add a comment.

Reply:
I also makes it possible to disable this benchmark if C++17 is unavailable. Note that our 'tools' already require C++17. But this is fine because we only require C++17 for optional components.